docs/src/tutorials/tutorial_2.md

```diff
@@ -1,5 +1,367 @@
+
+```julia
+using Javis
+using Random
+```
+
+From there, we need to define colors to represent no, low, medium, and high brain activity.
+Feel free to change up the colors!
+I chose these colors that need to be added to your code:
+
+```julia
+indicators = ["tomato", "darkolivegreen1", "gold1", "white"]
+```
+
+From there, we also need to change the code block that defined the electrode locations.
+The previous electrode code looked like this
+
+```julia
+...
+        Action(
+            :electrodes,
+            (args...) ->
+                electrode.(
+                    electrode_locations,
+                    "white",
+                    "black",
+                    :fill,
+                    radius,
+                    electrode_names,
+                ),
+        ),
+...
+```
+
+However, what we now need to change is `"white"` to `rand(indicators)` for each electrode.
+An example resulting electrode would look like this:
+
+```julia
+...
+        Action(
+            :electrodes,
+            (args...) ->
+                electrode.(
+                    electrode_locations,
+		    rand(indicators),
+                    "black",
+                    :fill,
+                    radius,
+                    electrode_names,
+                ),
+        ),
+...
+```
+
+Once all these modifications were made, execute your eeg code and you should get something that looks like this:
+
+![](assets/eeg_colors.gif)
+
+IT'S ALIVE!!! 🔬
+We could finish this now, but let's add just a little bit more polish to it.
+
+# As You Can See Here...
+
+Let's add some information to our animation. 
+We can create an info box using the following function:
+
+```julia
+function info_box(video, action, frame)
+    fontsize(12)
+    box(140, -210, 170, 40, :stroke)
+    text("10-20 EEG Array Readings", 140, -220, valign = :middle, halign = :center)
+    text("t = $(frame)s", 140, -200, valign = :middle, halign = :center)
+end
+```
+
+It's invocation in the code looks like this:
+
+```julia
+...
+Action(:same, :info, info_box),
+...
+```
+
+> **NOTE:** The function for `info_box` is a little different!
+> Each `Action` inside of the `javis` function automatically receives three objects being a `Video` object, which was previously defined outside of the `javis` function, the function, and the current frame number. 
+
+Once everything is executed, we get this very nice and clean looking animation which shows what this animation is and when parts of the brain are activated:
+
+![](assets/eeg.gif)
+
+# Conclusion
+
+Congratulations! 🎉 🎉 🎉
+You made a brain! 
+To recap, by working through this animation you should now:
+
+1. Clearly understand how to use an `Action` and how to make your own `Action`
+2. See how to create complex animations
+3. Know how to display meaningful information on your animations
+
+Great job leveling up your `Javis` skills! 💪
+
+---
+---
+
+> **Author(s):** Jacob Zelko, Ole Kröger \
+> **Date:** August 11th, 2020 \
```

Comment:
14th?

Shall we add this in every tutorial? Do we need that?

Reply:
I would argue yes because the elastic search engine would pull up the tutorial based on these tags. What do you think?